server/service/osquery_utils/queries.go

```diff
@@ -1281,6 +1284,36 @@ func sanitizeSoftware(h *fleet.Host, s *fleet.Software, logger log.Logger) {
 				s.Version = "20" + s.Version // Cloudflare WARP was released on 2019.
 			},
 		},
+		{
+			checkSoftware: func(h *fleet.Host, s *fleet.Software) bool {
+				return h.Platform == "windows" && citrixName.Match([]byte(s.Name))
+			},
+			mutateSoftware: func(s *fleet.Software) {
```

Comment:
Let's support the case where Citrix Workspace fixes their versioning to report the sanitized version `2309.1.104`.

Reply:
+1 and a test

Reply:
https://github.com/fleetdm/fleet/pull/15152/files#diff-9aebb07c5e9eae43b4cf167ef1ac12d720a613da219e5342d9a8e992e5fd0815R1298-R1301
these lines are for that, there's an additional test I added as well @mostlikelee